spec/models/user_spec.rb

```diff
@@ -759,16 +765,16 @@
       end
     end
 
-    describe '#mail_confirm_email' do
+    describe '#send_confirm_email' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.